test/fakeintake/server/server_test.go

```diff
@@ -28,22 +28,29 @@ import (
 )
 
 func TestServer(t *testing.T) {
+	for _, driver := range []string{"memory", "sql"} {
+		testServer(t, WithStoreDriver(driver))
+	}
+}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**